GitCommands/RevisionReader.cs

```diff
@@ -338,10 +330,11 @@ int CountParents(int baseOffset)
             #region Timestamps
 
             // Lines 2 and 3 are timestamps, as decimal ASCII seconds since the unix epoch, each terminated by `\n`
-            var authorDate = ParseUnixDateTime();
-            var commitDate = ParseUnixDateTime();
+            int startIndex = offset;
```

Comment:
`startIndex` is not used, is it?

Reply:
removed
(I had implemented comparing the time slices to optimize, removed it to keep complexity down but missed this)